x-pack/plugin/core/src/main/resources/logs-default-mappings.json

```diff
@@ -0,0 +1,152 @@
+{
+  "template": {
+    "mappings": {
+      "_meta": {
+      },
+      "dynamic_templates": [
+        {
+          "strings_as_keyword": {
+            "mapping": {
+              "ignore_above": 1024,
+              "type": "keyword"
+            },
+            "match_mapping_type": "string"
+          }
+        }
+      ],
+      "date_detection": false,
+      "properties": {
```

Comment:
We need to discuss, if dynamic mapping in here should be true or false. Historically I would have said true but seeing that in the future we will also have the possibility to query some fields which are not indexed, I wonder if `false` would be better here?

Reply:
I think we should leave dynamic mapping enabled, especially since we're dropping other fields (things like `host.hostname`) that a user may still want to query